roles/ceph-dashboard/tasks/configure_dashboard.yml

```diff
@@ -5,10 +5,18 @@
   when: containerized_deployment | bool
 
 - name: disable SSL for dashboard
-  command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} config set mgr mgr/dashboard/ssl false"
-  delegate_to: "{{ groups[mon_group_name][0] }}"
-  run_once: true
   when: dashboard_protocol == "http"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  block:
+   - name: get SSL status for dashboard
```

Comment:
shouldn't this be executed only once time as well?

Reply:
Yep, you are right.